packages/caliper-ethereum/lib/ethereum.js

```diff
@@ -212,6 +222,21 @@ class Ethereum extends BlockchainInterface {
                 context.nonces[context.fromAddress] = nonce + 1;
                 params.nonce = nonce;
             }
+
+            if (context.gasPrice !== undefined) {
```

Comment:
In retrospect this block doesn't need to be an if statement, as we'd just be assigning `undefined` if it were eliminated. It's benign to leave it in, but I'm happy to remove it if it makes people happy. Same comment for line 233.